Dockerfile

```
@@ -1,11 +1,20 @@
 FROM node:14.15-alpine AS BUILD_IMAGE
 
+ARG TARGETPLATFORM
+ENV TARGETPLATFORM=${TARGETPLATFORM:-linux/amd64}
+
 ARG COMMIT_TAG
 ENV COMMIT_TAG=${COMMIT_TAG}
 
 COPY . /app
 WORKDIR /app
 
+RUN \
+  case "${TARGETPLATFORM}" in \
+    'linux/arm64') apk add --update --no-cache python make g++ ;; \
+    'linux/arm/v7') apk add --update --no-cache python make g++ ;; \
```

```suggestion
    'linux/arm64') apk add --no-cache python make g++ ;; \
    'linux/arm/v7') apk add --no-cache python make g++ ;; \
```
I was previously advised that the `--update` flag is not desirable if you want consistently reproducible builds.

---

IIRC `--no-cache` implies `--update`, so the package cache is up-to-date for the packages you want to install.

I'll remove it since it's redundant.

---

@onedr0p Ahhh you're right 🤔🤷🏻‍♀️